src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -662,8 +662,19 @@ class DocBaseViewer extends BaseViewer {
                 // eslint-disable-next-line
                 console.error(err);
 
+                // pdf.js gives us the status code in their error message
+                const { status } = err;
+
                 // Display a generic error message but log the real one
-                const error = new PreviewError(ERROR_CODE.CONTENT_DOWNLOAD, __('error_document'), {}, err.message);
+                const error =
+                    status === 202
```

Comment:
Does pdf.js recognize 202 as an error code or is this placeholder?

Reply:
it's recognized as a success, but then PDF.js will throw a `MissingPDFException`, which includes the HTTP status code that led to the error.